WebContent/js/components/Setting.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2016, 2019
+ */
+
+import React from 'react';
+import Switch from '@material-ui/core/Switch';
+import FormControl from '@material-ui/core/FormControl';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import styled from 'styled-components';
+import { ENABLE_REDUX_LOGGER, NOTIFICATION_DURATION, getStorageItem, setStorageItem } from '../utilities/storageHelper';
+
+
+const Settings = styled.div`
```

Comment:
Also it would be good to do additional UI/UX review, maybe later.

- Common text/brand colours better to be defined in one place.
- SettingSection grid template now uses only 1/3 of space, but if it is just a template for other settings in future, then it's fine.
`grid-template-columns: 33% 33% 33%`
- h5 header with '*require reload' text should be a footer instead, from my point of view.

Reply:
yes, branding we can centralize, made it one column 100%, will exapnd when more settings, h5 moved to below